hw02_unpack_string/unpack_test.go

```diff
@@ -17,10 +17,10 @@ func TestUnpack(t *testing.T) {
 		{input: "", expected: ""},
 		{input: "aaa0b", expected: "aab"},
 		// uncomment if task with asterisk completed
```

Comment:
Тут можно было бы добавить тестов c unicode символами вроде вот такого
```suggestion
		// uncomment if task with asterisk completed
		{name: "nonASCII: emoji ", input: "♫♫♫0", expected: "♫♫"},
		{name: "nonASCII: emoji", input: "🙂9", expected: "🙂🙂🙂🙂🙂🙂🙂🙂🙂"},
```